src/libasr/codegen/asr_to_c.cpp

```diff
@@ -806,8 +806,9 @@ R"(
         std::string idx = std::move(src);
         this->visit_expr(*x.m_arg);
         std::string str = std::move(src);
+        idx = "((" + idx + " - 1) >= 0 ? (" + idx + " - 1) : (strlen(" + str + ") + " + idx + " - 1))";
```

Comment:
This change should not be done, as it slows down every index access, see above.

Reply:
Removed this. Thanks!